recipes/pykdtree/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "1.1.1" %}
+
+package:
+    name: pykdtree
+    version: {{ version }}
+
+source:
+    fn: pykdtree-{{ version }}.tar.gz
+    url: https://pypi.python.org/packages/source/p/pykdtree/pykdtree-{{ version }}.tar.gz
+    md5: 936b0844b3d06d8f2f2a1ca40f587cfb
+
+build:
+    number: 0
+    skip: True  # [win]
+
+requirements:
+    build:
+        - python
+        - setuptools
+        - numpy x.x
+        - gcc  # [osx]
+    run:
+        - python
+        - numpy x.x
+        - libgcc  # [osx]
+
+test:
+    imports:
+        - pykdtree
+
+about:
+    home: https://pypi.python.org/pypi/pykdtree
+    license: GPL-3.0
```

Comment:
Since they don't ship the license file that is hard to determine... Still. I am OK changing to LGP3, but that is an upstream problem IMO.